app/views/foreman_wreckingball/hosts/status_dashboard.html.erb

```diff
@@ -7,3 +10,19 @@
     render 'status_dashboard_empty'
   end
 %>
+<div id="confirmation-modal" class="modal fade">
```

Comment:
Can we reuse something like that?

https://github.com/theforeman/foreman/blob/33d9f9eeec63da0322b7d318077c2a7e58ffff11/app/views/smart_proxies/logs/_modal.html.erb